WebApplication/Processing/Arranger.cs

```diff
@@ -46,6 +48,7 @@ public class Arranger
         public readonly string OutputSAT = $"{Guid.NewGuid():N}.sat";
         public readonly string OutputRFA = $"{Guid.NewGuid():N}.rfa";
         public readonly string BomJson = $"{Guid.NewGuid():N}.bom.json";
+        public readonly string DrawingVieawbles = $"{Guid.NewGuid():N}.drawing.pdf";
```

Comment:
> `DrawingVieawbles`

Typo (should be `DrawingViewables`)
// there are several places to change, so I'm afraid to use "Suggestion" feature here